apps/accounts/serializers.py

```diff
@@ -37,3 +54,36 @@ def update(self, instance, validated_data):
             profile.affiliation = affiliation
             profile.save()
         return instance
+
+
+class UserInvitationSerializer(serializers.ModelSerializer):
+    """
+    Serializer to store the invitation details
+    """
+
+    challenge_title = serializers.SerializerMethodField()
+    challenge_host_team_name = serializers.SerializerMethodField()
+    username = serializers.SerializerMethodField()
```

Comment:
Instead of storing the username, you should define a UserSerializer that uses the 'User' model.